libs/components/layout/src/lib/modules/action-button/action-button-container.component.ts

```diff
@@ -44,54 +44,66 @@ export class SkyActionButtonContainerComponent
    */
   @Input()
   public set alignItems(value: SkyActionButtonContainerAlignItemsType) {
-    this._alignItems = value;
+    this.#_alignItems = value;
   }
 
   public get alignItems(): SkyActionButtonContainerAlignItemsType {
-    return this._alignItems || 'center';
+    return this.#_alignItems || 'center';
```

Comment:
Any getters should be changed to have their logic in the setter instead. We shouldn't be doing logic in getters.

Reply:
fixed